lib/plaid/connection.rb

```diff
@@ -9,22 +9,29 @@ class << self
       def post(path, options = {})
         uri = build_uri(path)
         options.merge!(client_id: Plaid.customer_id, secret: Plaid.secret)
-        res = Net::HTTP.post_form(uri,options)
+        http = Net::HTTP.new(uri.host, uri.port)
+        http.use_ssl = (uri.scheme == 'https')
```

Comment:
Hello Rilian,

Yes, but in Plaid::Connection.patch, Plaid::Connection.delete, master current code uses :use_ssl => uri.scheme == 'https' check.

>         Net::HTTP.start(uri.hostname, uri.port, :use_ssl => uri.scheme == 'https') { |http| http.request(req) }
> res = Net::HTTP.start(uri.hostname, uri.port, :use_ssl => uri.scheme == 'https') { |http| http.request(req) }

So I also added ssl check in post & get method while doing modification and not used  strictly true.

Thanks,
Nitesh